main/src/ca/uwaterloo/flix/language/phase/Regions.scala

```diff
@@ -279,6 +279,9 @@ object Regions {
     case Expr.FixpointProject(_, exp, tpe, _, loc) =>
       visitExp(exp) ++ checkType(tpe, loc)
 
+    case Expr.InvokeMethod2(_, _, _, _, _, _) =>
+      Nil
```

Comment:
Recursive visit

Reply:
It seems like exps is not visited in Expr.Apply, I followed the same pattern for InvokeMethod2.

Reply:
That seems like a bug!!

Reply:
I see. Then would this be good for both of them ? :
`exps.flatMap(visitExp) ++ visitExp(exp) ++ checkType(tpe, loc)`

Reply:
Yes